### DISCUSSION TITLE
Problems with Saturn rendering?

### DISCUSSION THREAD

**Discussion by tflaulhabe** [Stellarium/stellarium#1644] [Q&A] [answered]
Hey, I'm experiencing some bugs, maybe with the rendering of some objects? Saturn is the only one with this problem in Stellarium by now. I actually believe that's a problem with my computer, but I don't know how to solve it.
I wasn't experiencing this problem some weeks ago, so I don't know what could have happened.

I'm using Ubuntu 20.04.02 LTS (64bit), with

Intel® Celeron(R) CPU N2940 @ 1.83GHz × 4 
Intel® HD Graphics (BYT)
3,7 GiB of memory

There's a printscreen of my problem attached.

![Screenshot from 2021-05-07 10-17-54](https://user-images.githubusercontent.com/69877699/117455464-985bd680-af1d-11eb-90af-9f49b095fed9.png)

**Comment by alex-w** [comment, marked as answer]
Probably Uranus and Neptune will have same issue.

Please try run `LIBGL_ALWAYS_SOFTWARE=1 stellarium`

**Reply by tflaulhabe** [reply]
I've tried `LIBGL_ALWAYS_SOFTWARE=1 stellarium-daily.stellarium` and my problem with Saturn was solved. However, this is causing my Stellarium to work very slowly, in 3-4 FPS, where it previously ran with ~16 FPS with ease.

I don't know if it's a problem with my Ubuntu, since I'm observing this issue in other applications as well.

**Reply by alex-w** [reply]
Could you try install stellarium from our ppa https://launchpad.net/~stellarium ?

**Reply by alex-w** [reply]
This is probably driver issue, but without guarantees, because this video card has bad performance.

**Reply by gzotti** [reply]
If you call LIBGL_ALWAYS_SOFTWARE=1 you obviously switch off the graphics card and do all computation on the CPU. This is like "MESA MODE" in Windows. 
If this is a desktop PC, you could install some old Geforce (8xxx and later) which often have better drivers.

**Reply by alex-w** [reply]
> If this is a desktop PC, you could install some old Geforce (8xxx and later) which often have better drivers.

No, this is laptop :-/

**Reply by tflaulhabe** [reply]
> Could you try install stellarium from our ppa https://launchpad.net/~stellarium ?

Yeah, I've installed with `sudo apt install stellarium`, but the problem persists. 
I don't know why this problem has come up now, the rendering was alright last month.

Anyway, thanks for the possible solutions to solve it.
